truenas/webui
# NAS-121257 / 23.10 / F60 support for UI and mock utility

Test requirements: 

- Ensure product image is visible on main dashboard SysInfo widget
- Ensure product image is visible under system/general support card
- Make sure enclosure ui is available and shows visualization for F60

Do the above scenarios against real machine (check ticket for url)
Also try out mock utility...

`yarn ui mock` and select options "included" > "f60.config.json"
